src/index.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App';
+import BitcoinHeader from './components/bitcoin_header';
+import BitcoinContent from './components/bitcoin_content';
 import * as serviceWorker from './components/serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+class App extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
@louieSison716 you can replace the wrapping div with <React.Fragment></React.Fragment>

Reply:
@jemjemzms  okay, noted.